kcidb/orm/query.py

```diff
@@ -0,0 +1,1461 @@
+from kcidb.orm.__init__ import re, Type, LIGHT_ASSERTS, textwrap
```

Comment:
Same here: do not import anything from `__init__` explicitly. Also, definitely do not import common definitions available elsewhere (such as `LIGHT_ASSERTS`, or `textwrap`) via a third-party module.

Reply:
Okay sir.